src/PointwiseFunctions/AnalyticData/ForceFree/RotatingDipole.cpp

```diff
@@ -123,6 +123,35 @@ tuples::TaggedTuple<Tags::TildeQ> RotatingDipole::variables(
   return {make_with_value<Scalar<DataVector>>(coords, 0.0)};
 }
 
+tuples::TaggedTuple<Tags::NsInteriorMask> RotatingDipole::variables(
+    const tnsr::I<DataVector, 3, Frame::Grid>& grid_frame_coords,
+    tmpl::list<Tags::NsInteriorMask> /*meta*/) {
+  std::optional<Scalar<DataVector>> result{};
+
+  DataVector r_squared = get(dot_product(grid_frame_coords, grid_frame_coords));
+
+  // NS radius rescaled to 1.0 on grid.
```

Comment:
Is this true?  I don't see anything in the fields doing this, or is the NS region arbitrary?

Reply:
We set `c=1` in this evolution system so the length (=time) scale can be arbitrarily rescaled. I could add some option like `NsRadius`, but just thought it'd be convenient to just fix `R_* = 1.0` and just interpret all the results from this initial data in terms of `R_S`. For example, `t=1.0` in the code unit would correspond to `R_S / c` in physical unit.

Reply:
No that's fine.  I was expecting the fields to have a different form inside the star, but I see now that that isn't the case.